core/lib/transitions/simpleClassTransition.ts

```diff
@@ -2,14 +2,32 @@ import {createCSSTransition} from './cssTransitions';
 import {addClasses, reflow, removeClasses} from './utils';
 
 export interface SimpleClassTransitionConfig {
+	/**
+	 * a list of classes to add to the collapsable element while transitioning
+	 */
 	animationPendingClasses?: string[];
+	/**
+	 * a list of classes to add to the collapsable element while transitioning towards the show state
+	 */
 	animationPendingShowClasses?: string[];
+	/**
+	 * a list of classes to add to the collapsable element while transitioning towards the hidden state
```

Comment:
```suggestion
	 * a list of classes to add to the collapsable element while transitioning towards the hide state
```
(for consistency with the previous item, either `shown` and `hidden` or `show` and `hide`)